enrollment-server-api-model/pom.xml

```diff
@@ -33,6 +33,24 @@
         <version>1.4.0-SNAPSHOT</version>
     </parent>
 
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <version>${jakarta-validation-api.version}</version>
+            <scope>compile</scope>
```

Comment:
Default. Any reason to declare it explicitly?

Reply:
Just a copy-paste reason.